include/seqan3/alphabet/composite/alphabet_tuple_base.hpp

```diff
@@ -24,109 +24,58 @@
-                                       tuple_derived_t,
-                                       fun_t,
-                                       alphabet_tuple_base<tuple_derived_t, tuple_comps...>> = false;
-
-// guard against self (derived)
-template <typename ...tuple_comps,
-          typename tuple_derived_t,
-          template <typename> typename fun_t>
-inline bool constexpr one_component_is<alphabet_tuple_base<tuple_derived_t, tuple_comps...>,
-                                       tuple_derived_t,
-                                       fun_t,
-                                       tuple_derived_t> = false;
-
-// guard against types that have conversion operators to derived
-template <typename ...tuple_comps,
-          typename tuple_derived_t,
-          template <typename> typename fun_t,
-          typename other_t>
-    requires convertible_to_by_member<other_t, tuple_derived_t>
-inline bool constexpr one_component_is<alphabet_tuple_base<tuple_derived_t, tuple_comps...>,
-                                       tuple_derived_t,
-                                       fun_t,
-                                       other_t> = false;
-
-// guard against components
-template <typename ...tuple_comps,
-          typename tuple_derived_t,
-          template <typename> typename fun_t,
-          typename other_t>
-    requires type_in_pack_v<other_t, tuple_comps...>
-//     requires meta::in<recursive_tuple_components<alphabet_tuple_base<tuple_derived_t, tuple_comps...>>::type,
-//                       other_t>::value
-inline bool constexpr one_component_is<alphabet_tuple_base<tuple_derived_t, tuple_comps...>,
-                                       tuple_derived_t,
-                                       fun_t,
-                                       other_t> = false;
-
-// during comparisons, guard against types that could be converted to self (because that is preferred)
-// (may not be done during assignment or construction because of recursiveness)
-template <typename ...tuple_comps,
-          typename tuple_derived_t,
-          typename other_t>
-    requires implicitly_convertible_to<other_t, tuple_derived_t>
-inline bool constexpr one_component_is<alphabet_tuple_base<tuple_derived_t, tuple_comps...>,
-                                       tuple_derived_t,
-                                       seqan3::detail::weakly_equality_comparable_with_,
-                                       other_t> = false;
-template <typename ...tuple_comps,
-          typename tuple_derived_t,
-          typename other_t>
-    requires implicitly_convertible_to<other_t, tuple_derived_t>
-inline bool constexpr one_component_is<alphabet_tuple_base<tuple_derived_t, tuple_comps...>,
-                                       tuple_derived_t,
-                                       weakly_ordered_with_,
-                                       other_t> = false;
-//!\endcond
+template <typename tuple_derived_t, typename t>
+    requires requires
```

Comment:
doxygen `//!\cond` guard?

Reply:
It's inside the macro scope so doxygen can't see anyway.